### PR TITLE
Fix Docker dev setup and upgrade Node

### PR DESCRIPTION
**Pull request by taneliang**
* Upgrade Node in Dockerfiles from Node 10 LTS to Node 12 LTS.
* Remove unused files
* Fix symbol not found Chromium error in /export by not using
  chromium@edge
* Disable Nodemon upgrade notification in /export
* Remove ES/Kibana proxying in dev, since we're just using Elastic Cloud
* Remove short-url.php in dev since it's a prod detail
* Remove unused start-prod.sh and newly unused proxy.example.env

### CODE REVIEW

File: export/Dockerfile.dev [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
+FROM node:12-alpine
 
-# Installs latest Chromium (71) package.
+# Installs Chromium package.
 RUN apk update && apk upgrade && \
     echo @edge http://nl.alpinelinux.org/alpine/edge/community >> /etc/apk/repositories && \
     echo @edge http://nl.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories && \
     apk add --no-cache \
-      chromium@edge \
-      nss@edge \
```
Comment (li-kai) [inline export/Dockerfile.dev]: If I wasn't wrong, these dependencies exist to fix font rendering. Could you check if timetable download fonts are working fine?

File: website/Dockerfile.dev [hunk 2]
```diff
@@ -1,4 +1,8 @@
-FROM node:10-alpine
+FROM node:12-alpine
+
+RUN apk update && \
+    apk add --no-cache git python build-base && \
+    mkdir -p /home/node/app
 
 RUN mkdir -p /home/node/app
```
Comment (li-kai) [inline website/Dockerfile.dev]: This statement seems redundant

File: export/Dockerfile.dev (hunk 1 above)
Comment (taneliang) [reply export/Dockerfile.dev]: Just checked; the fonts look fine. I think you previously had to install those for compatibility with `chromium@edge`

**Comment by taneliang** [comment]
CI actually passed (https://circleci.com/workflow-run/333f81dc-ac69-4fe3-b5a7-26864b8794e0), but it was a rerun. Status check is showing a cancelled rerun of this rerun. Merging PR.

**Pull request merged**